tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -102,8 +110,11 @@ def retrieve_user_im(self, email_addresses):
                     list[address] = self.caching.get(address)
                     continue
 
-            response = self.client.api_call(
-                "users.lookupByEmail", email=address)
+            response = requests.post(
```

Comment:
can you also remove the slack client import and instantiation since its not being used.